Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "gateway-server"
+version = "0.1.0"
```

Comment:
Unfortunately I don't remember how is it working for cargo crates, but we should be taking a version from a git tag - https://github.com/githedgehog/gateway-proto/blob/master/hack/tools.just#L7 (or pass it from the just when calling cargo). It's okay to keep it hardcoded for now, but will be a problem in the future.

Reply:
It seems versioning for Rust crates is done in commonly used `major.minor.patch` approach